Source/WebCore/platform/graphics/ca/win/PlatformCAAnimationWin.cpp

```diff
@@ -539,13 +539,12 @@ void PlatformCAAnimationWin::copyKeyTimesFrom(const PlatformCAAnimation& value)
 
 void PlatformCAAnimationWin::setTimingFunctions(const Vector<Ref<const TimingFunction>>& value, bool reverse)
 {
-    UNUSED_PARAM(reverse);
     if (animationType() != Keyframe)
         return;
 
     RetainPtr<CFMutableArrayRef> array = adoptCF(CFArrayCreateMutable(0, value.size(), &kCFTypeArrayCallBacks));
-    for (size_t i = 0; i < value.size(); ++i)
-        CFArrayAppendValue(array.get(), toCACFTimingFunction(value[i].ptr(), reverse).get());
+    for (auto& timingFunction : value)
+        CFArrayAppendValue(array.get(), toCACFTimingFunction(timingFunction.get(), reverse).get());
```

Comment:
I feel like there is an existing utility function to convert `Vector's` of `CFThings` to a `CFArray` of `CFThings`, but I can't find it, so this looks fine to me.

Reply:
You're probably thinking of `createNSArray` from `<wtf/VectorCocoa.h>`, but there is no `createCFArray`, and since this is Windows C++ code, we can't use Objective-C.

Honestly, I find the syntax for `createNSArray` pretty hard to read (but maybe that's just because I'm not use to reading it frequently).  But `toCACFTimingFunction()` is called in one other place where there is no `Vector` of objects in use, so the conversion code would have to be duplicated or extracted anyway, so I really don't see the advantage here.